susemanager-utils/susemanager-sls/susemanager-sls.changes

```diff
@@ -1,6 +1,7 @@
 -------------------------------------------------------------------
 Mon Aug 09 11:11:17 CEST 2021 - jgonzalez@suse.com
 
+- Enable basic support for Debian 11
```

Comment:
Place the changelog this line on top, and since it's the first one after the last release, add an empty line below.

It needs to look like this:
```
- Enable basic support for Debian 11
- 
-------------------------------------------------------------------
Mon Aug 09 11:11:17 CEST 2021 - jgonzalez@suse.com
```

Same for the other changelog entries :-)

Reference: https://github.com/uyuni-project/uyuni/wiki/Contributing#git-changelogs